src/glass/src/app/core/dashboard/widgets/volumes-dashboard-widget/volumes-dashboard-widget.component.ts

```diff
@@ -4,7 +4,10 @@ import { Observable } from 'rxjs';
 
 import { AbstractDashboardWidget } from '~/app/core/dashboard/widgets/abstract-dashboard-widget';
 import { DatatableColumn } from '~/app/shared/models/datatable-column.type';
-import { OrchService, Volume } from '~/app/shared/services/api/orch.service';
+import {
```

Comment:
Needs to be applied to the https://github.com/aquarist-labs/aquarium/blob/main/src/glass/src/app/core/dashboard/widgets/volumes-dashboard-widget/volumes-dashboard-widget.component.spec.ts as well, e.g.:

```
diff --git a/src/glass/src/app/core/dashboard/widgets/volumes-dashboard-widget/volumes-dashboard-widget.component.spec.ts b/src/glass/src/app/core/dashboard/widgets/volumes-dashboard-widget/volumes-dashboard-widget.component.spec.ts
index 2513220..d27c677 100644
--- a/src/glass/src/app/core/dashboard/widgets/volumes-dashboard-widget/volumes-dashboard-widget.component.spec.ts
+++ b/src/glass/src/app/core/dashboard/widgets/volumes-dashboard-widget/volumes-dashboard-widget.component.spec.ts
@@ -7,7 +7,7 @@ import { of } from 'rxjs';
 
 import { DashboardModule } from '~/app/core/dashboard/dashboard.module';
 import { VolumesDashboardWidgetComponent } from '~/app/core/dashboard/widgets/volumes-dashboard-widget/volumes-dashboard-widget.component';
-import { OrchService, Volume } from '~/app/shared/services/api/orch.service';
+import { LocalNodeService, Volume } from '~/app/shared/services/api/local.service';
 
 describe('DevicesDashboardWidgetComponent', () => {
   let component: VolumesDashboardWidgetComponent;
@@ -202,7 +202,7 @@ describe('DevicesDashboardWidgetComponent', () => {
   beforeEach(() => {
     fixture = TestBed.createComponent(VolumesDashboardWidgetComponent);
     component = fixture.componentInstance;
-    spyOn(TestBed.inject(OrchService), 'volumes').and.returnValue(of(mockedVolumes));
+    spyOn(TestBed.inject(LocalNodeService), 'volumes').and.returnValue(of(mockedVolumes));
     fixture.detectChanges();
   });
```

Reply:
thanks! will do.